install_config/storage_examples/direct_storage.adoc

```diff
@@ -0,0 +1,481 @@
+and create the persistent volume:
+
+====
+----
+# oc create -f nfs-pv.yaml 
+persistentvolume "nfs-pv-prebind" created
+----
+====
+
+Verify that the persistent volume was created:
+
+====
+----
+# oc get pv
+NAME                     LABELS    CAPACITY   ACCESSMODES   STATUS      CLAIM                         REASON    AGE
+nfs-pv-prebind           <none>    1Gi        RWX           Available                                           4s
+----
+====
+
+=== 2. Creating the Persistent Volume Claim (PVC)
+In general a persistent volume claim (PVC) specifies the desired access mode and storage capacity.
+Currently, based on only these two attributes, a PVC is bound to a single PV. However in this example,
+we pre-bound the PVC to our PV we created.
+
+.PVC Object Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: nfs-claim-prebind  <1>
+spec:
+  accessModes:
+  - ReadWriteMany      <2>
+  resources:
+     requests:
+       storage: 1Gi    <3>
+----
+<1> The claim name referenced by the PV spec definition ClaimRef Name.
+<2> As mentioned above for PVs, the `*accessModes*` do not enforce access right,
+but rather act as labels to match a PV to a PVC.
+<3> This claim will look for PVs offering *1Gi* or greater capacity.
+====
+
+Save the PVC definition to a file, for example *_nfs-pvc.yaml_*,
+and create the PVC:
+
+====
+----
+# oc create -f nfs-pvc.yaml 
+persistentvolumeclaim "nfs-claim-prebound" created
+
+#and verify the PVC was created and bound to the expected PV:
+# oc get pvc
+NAME                       LABELS    STATUS    VOLUME               CAPACITY   ACCESSMODES   AGE
+nfs-claim-prebound         <none>    Bound     nfs-pv-prebind       1Gi        RWX           24s
+                                    <1>
+----
+<1> the claim (nfs-claim-prebind) was bound to the "nfs-pv-prebind" PV.
+====
+
+=== 3. Creating the Pod
+A pod definition file or a template file can be used to define a pod. Below is a pod spec that
+creates a single container and mounts the nfs volume for read-write access:
+
+.Pod Object Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx-nfs-pod <1>
+  labels:
+    name: nginx-nfs-pod
+spec:
+  containers:
+    - name: nginx-nfs-pod
+      image: fedora/nginx <2>
+      ports:
+        - name: web
+          containerPort: 80
+      volumeMounts:
+        - name: nfsvol <3>
+          mountPath: /usr/share/nginx/html <4>
+  securityContext:
+      supplementalGroups: [100003] <5>
+      privileged: false
+  volumes:
+    - name: nfsvol
+      persistentVolumeClaim:
+        claimName: nfs-claim-prebind
+----
+<1> The name of this pod as displayed by `oc get pod`.
+<2> The image run by this pod.
+<3> The name of the volume. This name must be the same in both the `containers` and `volumes` sections.
+<4> The mount path as seen in the container.
+<5> The group id to be assigned to the container.
```

Comment:
There has been, up to this point, no mention of group id 100003 so it seems "out of the blue" here.